lib/phoenix/router.ex

```diff
@@ -586,6 +586,7 @@ defmodule Phoenix.Router do
     * `:host` - a string containing the host scope, or prefix host scope, ie
                 `"foo.bar.com"`, `"foo."`
     * `:private` - a map of private data to merge into the connection when a route matches
+    * `:assigns` - a map of data to merge into the connection when a route matches
```

Comment:
we need to clarify which takes precedence. Something like, "The default map of assigns for the connection"